ci-operator/config/openshift/etcd-operator/openshift-etcd-operator-openshift-4.1.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  base:
+    name: "4.1"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+canonical_go_repository: github.com/coreos/etcd-operator
+images:
+- dockerfile_path: Dockerfile.openshift
+  from: base
+  to: etcd-operator
+promotion:
+  name: "4.1"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.1"
+  namespace: ocp
+tests:
+- as: unit
+  commands: |
+    curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
Let's add `dep` to the installs in the base image(s) for recent Go versions so you don't need to do this every test

Reply:
https://github.com/openshift/release/blob/master/projects/origin-release/golang-1.10/Dockerfile#L32